app/src/main/java/com/sunlightlabs/android/congress/fragments/LegislatorListFragment.java

```diff
@@ -271,7 +271,7 @@ public void onListItemClick(ListView parent, View v, int position, long id) {
 	}
 
 	public void selectLegislator(Legislator legislator) {
-		startActivity(Utils.legislatorIntent(getActivity(), legislator));
+		startActivity(Utils.legislatorIntent(legislator.bioguide_id));
```

Comment:
This causes the legislator home screen to go make another API call when coming in from a list view, rather than reuse the downloaded data we got from the list view, since the PP API doesn't give the same breadth of fields for the list as for the fetch.